packages/wallets/fractal/package.json

```diff
@@ -33,11 +33,11 @@
         "@solana/web3.js": "^1.58.0"
     },
     "dependencies": {
-        "@fractalwagmi/solana-wallet-adapter": "0.0.9",
+        "@fractalwagmi/solana-wallet-adapter": "0.0.12",
```

Comment:
Can this get bumped up to a minor version, and use `^` for versioning here instead of an exact version? It would be great to not have to release this package, and the `wallets` package, every time there's an update to a dependency.

Reply:
Done.